workshops/responsive_navbar/README.md

```diff
@@ -0,0 +1,509 @@
+description: 'Build a fully responsive navbar for your websites.'
+author: '@faisalsayed10'
+---
+
+# Rebar
+
+You probably must have built navbars in your websites. It is a challenging task for many of us to make a good and responsive navbar. Well, today we'll be building a fully responsive navbar! It is not only easy to build but also very smooth and beautiful!
+
+But first, Let's have a look on various navbars on the internet:
+
+The responsive nav:
+
+[![The responsive nav](https://cloud-hskw22p2h.vercel.app/hackclub-nav.gif)](https://hackclub.com)
+
+The simple nav:
+
+[![The simple nav](https://cloud-o35ba6qxw.vercel.app/fayd-nav.gif)](https://fayd.me)
+
+Or even this?
+
+![The weirdo nav](https://cloud-qiqy4s3qm.vercel.app/worst-nav.gif)
+
+And this is what we'll be building today!
+
+Here's a sneak peek:
+
+[![sneak peek](https://cloud-kezdm1i9q.vercel.app/workshop-nav.gif)](https://responsive-navbar.faisalsayed1.repl.co/)
+
+Here's the [live demo](https://responsive-navbar.faisalsayed1.repl.co/).  
+Here's the [source code](https://repl.it/@FaisalSayed1/Responsive-Navbar).
+
+## Part 1: Prerequisites
+
+You should have a beginner understanding of:
+
+- HTML
+- CSS
+- JavaScript
+
+## Part 2: Setup
+
+### Setting up your code environment on Repl.it
+
+[Repl.it](https://repl.it) is an online code editor. You don't have to use Repl.it but I suggest you do as it sets everything up for you and you don't require any installations.
+
+To get started, go to [https://repl.it/languages/html](https://repl.it/languages/html). Your coding environment will spin up in just a few seconds!
+
+You should see something like the following:
+
+![Starting View](https://cloud-p7qnbqzo6.vercel.app/image.png)
+
+## Part 3: Building the project
+
+### 1) HTML
+
+Let's start writing the HTML code required in the `index.html` file. We'll write all our code inside the `<body></body>` tags.
+
+**NOTE:** Make Make sure the `<script>` tag, which is in your body, is just above the closing `</body>` tag and whatever code you write in the `<body>` should be above that `<script>` tag or else it will throw you errors.
+
+First, we'll build a simple structure of the Navbar:
+
+```html
+<body>
+  <nav class="navbar">
+    <ul class="elements">
+      <li class="nav-el">HOME</li>
+      <li class="nav-el">ABOUT</li>
+      <li class="nav-el">CONTACT</li>
+      <li class="nav-el">PROJECTS</li>
+    </ul>
+  </nav>
+  <script src="script.js"></script>
+</body>
+```
+
+Explanation: In the `<nav>` element, we create an unordered list (`<ul>`) with the list items (`<li>`) as navlinks.
+
+Also, the `<nav>` element has a class of `navbar`, the `<ul>` has a class of `elements`, the `<li>` has a class of `nav-el` and as always, the  `<script>` tag is below all these lines.
+
+Now, we will also create a burger button which will only be visible for smaller screens and it will make the navbar pop up from the side. It will come below the unordered list inside the `<nav>` element.
```

Comment:
```suggestion
Now, we will also create a burger button that will only be visible for smaller screens and it will make the navbar pop up from the side. It will come below the unordered list inside the `<nav>` element.
```

Reply:
I think 'which' suits better here.